tools/daca2-download.py

```diff
@@ -117,8 +117,8 @@ def removeLargeFiles(path):
                 os.remove(g)
 
             # remove non-source files
-            elif g[-2:] not in {'.C', '.c', '.H', '.h'} and g[-3:] != '.cc' and\
-                    g[-4:] not in {'.cpp', '.cxx', '.c++', '.hpp', '.tpp', '.t++'}:
+            elif g[-2:] not in {'.C', '.c', '.H', '.h'} and g[-3:] not in { '.cc', '.hh' } and\
+                    g[-4:] not in {'.cpp', '.cxx', '.c++', '.hpp', '.hxx', '.tpp', '.txx'}:
```

Comment:
Why did you remove `.t++`? Where do these `.t*` file extensions come from? I have never seen them. But I guess there is a reason that they were added.
In this [StackOverflow question and answers](https://stackoverflow.com/questions/1545080/c-code-file-extension-cc-vs-cpp/1545085) are lists of used file extensions. From what I see we are still missing `.cp` for example. CodeWarrior seems to use or at least knows it.